parser/transcript/src/main/java/de/danoeh/antennapod/parser/transcript/PodcastIndexTranscriptParser.java

```diff
@@ -0,0 +1,23 @@
+package de.danoeh.antennapod.parser.transcript;
+
+import org.apache.commons.lang3.StringUtils;
+
+import de.danoeh.antennapod.model.feed.Transcript;
+
+public class PodcastIndexTranscriptParser {
```

Comment:
I think this branch is not up-to-date with the changes on the `transcript` branch. This file here is called just `TranscriptParser` over at the other branch